VultisigApp/VultisigApp/Views/Swap/SwapCryptoDetailsView.swift

```diff
@@ -147,16 +144,6 @@ struct SwapCryptoDetailsView: View {
         .opacity(swapViewModel.validateForm(tx: tx) ? 1 : 0.5)
         .padding(40)
     }
```

Comment:
This logic was added to avoid duplication of assets on Swap screen. removing this brings it back.

![Simulator Screenshot - iPhone 15 Pro - 2024-06-05 at 15 38 00](https://github.com/vultisig/vultisig-ios/assets/13535088/b54bd022-b3ae-4154-844a-8dc0f44742d8)
![Simulator Screenshot - iPhone 15 Pro - 2024-06-05 at 15 38 00](https://github.com/vultisig/vultisig-ios/assets/13535088/b54bd022-b3ae-4154-844a-8dc0f44742d8)